Security_Analytics/auditd_analysis/README.md

```diff
@@ -0,0 +1,174 @@
+# Alerting on Auditd CEF Data
+
+This **Getting Started with Elastic Stack** example provides sample files to ingest, analyze and alert on **Auditd Logs in the CEF Format** using the Elastic Stack. 
+
+Included are example Watches for proactively monitoring this data for possible security incidents.  These examples support the Security Analytics blog post series, specifically:
+ 
+[Integrating Elasticsearch with ArcSight SIEM - Part 5](https://elastic.co/blog/integrating-elasticsearch-with-arcsight-siem-part-5).
+
+This example complements the above blog post, providing a means to detect unusual processes starting on servers in the last N minutes. Unusual is defined as "A new process signature that has not occurred on the server historically".
+
+This example relies on the user creating a `process_signature` which acts as a unique identifier for a process. This is an imprecise process, relying on the user selecting fields to concatenate which best represent a unique identifier across server instances when combined.  Too coarse identifier, which fails to encode sufficient properties of the processes characteristics, will result in the alert failing to detect new process effectively. Too granular and all processes will incorrectly be identified as being new.
+
+For auditd CEF encoded data we utilise the following fields, concatenating them to form a single process_signature field with the delimiter “|” using a Logstash filter:
+
+- ahost - the host from which the event originated
+- ad.a0 - first argument of the command used to execute the process 
+- ad.a1 - second argument of the command. This could be removed for coarsening process detection.
+
+
+This example includes:
+
+- [`auditd.cef`](https://raw.githubusercontent.com/elastic/examples/master/Security_Analytics/audidt_analysis/audit.cef) - Sample Auditd logs in CEF format used in the above blog post.
+- `auditd_analysis_logstash.conf` - An appropriate Logstash configuration for indexing the above CEF data and creating a `process_signature` field.
+- `auditd_analysis_kibana.json` - Simple Kibana visualizations and dashboards for the associated blog posts
+- `unusual_process.json` -  A watch that detects new processes starting. REFERENCE ONLY. 
+- `unusual_process.inline.json` - The above watch in an inline execution format so it can be used with the `simulate_watch.py` script and be executed over the full dataset.
+- `simulate_watch.py` - A convenience script to executes the above watch. In order to test this watch against the provided test data set this script which performs a “sliding window” execution of the watch. 
```

Comment:
"executes" should be "execute" ; "...set this script..."  should be "...set, this script..."